react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/JMOngoingConferenceModule.java

```diff
@@ -0,0 +1,69 @@
+package org.jitsi.meet.sdk;
+
+import android.app.Notification;
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.module.annotations.ReactModule;
+
+
+@ReactModule(name = JMOngoingConferenceModule.NAME)
+class JMOngoingConferenceModule
+    extends ReactContextBaseJavaModule {
+
+    public static final String NAME = "JMOngoingConference";
+
+    private static JMOngoingConferenceModule jmOngoingConferenceModuleInstance;
+
+    public JMOngoingConferenceModule(ReactApplicationContext reactContext) {
+        super(reactContext);
+        jmOngoingConferenceModuleInstance = this;
+    }
+
+    static JMOngoingConferenceModule getInstance() {
+        return jmOngoingConferenceModuleInstance;
+    }
+
+    Notification build() {
+        Context context = getReactApplicationContext().getCurrentActivity();
+
+        return RNOngoingNotification.buildOngoingConferenceNotification(context);
+    }
+
+    void create() {
```

Comment:
Ditto.

Reply:
So we need to keep this method here because this is a context module from which we need to get the activity.
If we do it in another place  error: cannot find symbol
        Context activityContext = context.getCurrentActivity();